nifi-commons/nifi-sensitive-property-provider/pom.xml

```diff
@@ -59,6 +59,40 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.18.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.azure</groupId>
+                    <artifactId>azure-core-http-netty</artifactId>
```

Comment:
With Netty excluded, it looks like `io.netty:netty-tcnative-boringssl-static` should also be excluded.

Reply:
Doing some testing with this exclusion, will test the full build, will include this exclusion if full build is good